tree/ntuple/v7/test/CMakeLists.txt

```diff
@@ -23,10 +23,23 @@ if(MSVC)
                                      ${CMAKE_CURRENT_BINARY_DIR}/libCustomStruct.dll)
 endif()
 
+ROOT_STANDARD_LIBRARY_PACKAGE(RXTuple
+  NO_INSTALL_HEADERS
+  HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/RXTuple.hxx
+  SOURCES RXTuple.cxx
+  LINKDEF RXTupleLinkDef.h
+  DEPENDENCIES RIO
+)
```

Comment:
Actually, we may not need this but `ROOT_GENERATE_DICTIONARY` is probably enough.

Reply:
When testing with `ROOT_GENERATE_DICTIONARY`, did you add the resulting `.cxx/.o` file to the test executable?